README.md

```diff
@@ -127,6 +127,8 @@ Transforms expect the following dimensions.
 * `MuLawDecode`: (channel, time) -> (channel, time)
 * `Resample`: (channel, time) -> (channel, time)
 
+Complex numbers are supported via tensors of dimension (..., 2), and provide `magphase` to convert such a tensor into its magnitude and phase, and also `complex_norm` and `angle`.
+
```

Comment:
maybe "its magnitude through `complex_norm` and phase through `angle`."

Reply:
I agree we can simplify this. Let's only talk about `complex_norm` and `angle` then. How does that sound?